src/commands/config-rewrite.json

```diff
@@ -1,7 +1,7 @@
 {
     "REWRITE": {
         "summary": "Rewrite the configuration file with the in memory configuration",
-        "complexity": "PATCH__TBD__46__",
+        "complexity": "O(1)",
```

Comment:
i suppose that since N is the number of total configs in redis, and it's constant and low, it's ok to list it as O(1).
so that's applicable on my comment above too, which makes N the number of requested configs?
i'm not an expert on that matter.. @itamarhaber ?

Reply:
oh ok just saw this comment

IIRC Itamar and I discussed it and since the number of configurations is indeed constant it shouldn't count in the big O notation